Monika After Story/game/script-stories.rpy

```diff
@@ -348,11 +412,555 @@ label mas_story_immortal_love:
     m "The husband had ordered many bouquets in advance to ensure that his beloved wife would continue to receive flowers long after his death."
     m "Speechless and stunned, the wife read the note attached to the bouquet."
     m "{i}My love for you is eternal.{/i}"
-    
     m 1dubsu "Ahh..."
     m 1eua "Wasn't that a touching story, [player]?"
     m 1hua "I thought it was really romantic."
     m 1lksdlb "But I don't want to think of either of us dying."
     m 1eua "At least the ending was very heartwarming."
     m 1hua "Thanks for listening~"
     return
+
+# Scary stories start here
+label mas_scary_story_setup:
+    show monika 1dsc
+    $ mas_temp_r_flag = mas_is_raining
+    $ scene_change = True
+    $ mas_is_raining = True
+    pause 1.0
+    $ mas_temp_m_flag = morning_flag
+    call spaceroom(start_bg="monika_gloomy_room")
+    $ morning_flag = True
+    #stop music fadeout 1.0
+    play background audio.rain fadein 1.0 loop
+    show vignette zorder 13
+#    $ songs.current_track = songs.FP_NO_SONG
+#    $ songs.selected_track = songs.FP_NO_SONG
+
+    $ HKBHideButtons()
+    #$ store.songs.enabled = False
+    python:
+        story_begin_quips = [
+            "Alright let's start the story.",
+            "Ready to hear the story?",
+            "Ready for story time?",
+            "Let's begin~",
+            "Let's begin then~"
+        ]
+        story_begin_quip=renpy.random.choice(story_begin_quips)
+    m 3eua "[story_begin_quip]"
+    m 1duu "Ahem."
+    return
+
+label mas_scary_story_cleanup:
+    python:
+        story_end_quips = [
+            "Scared, " + player + "?",
+            "Did I scare you, " + player + "?",
+            "How was it?"
+        ]
+        story_end_quip=renpy.random.choice(story_end_quips)

```

Comment:
can you just apply `renpy.substitute` to the selected quip, so the quips can use the regular renpy string formatting codes?


https://www.renpy.org/doc/html/other.html#renpy.substitute